src/Integration.Vsix/packages.lock.json

```diff
@@ -133,6 +133,18 @@
         "resolved": "3.3.0",
         "contentHash": "ZVBJ43/IxaAl5iX+OuuDLQ9tSNJaRlP85SUMAdiGoqEbyK/ZPfQmPfqSMZMFpucRlf8HgsEK/hR73DwpCmqcLA=="
       },
+      "SonarAnalyzer.CSharp": {
+        "type": "Direct",
+        "requested": "[8.52.0.60960, )",
+        "resolved": "8.52.0.60960",
+        "contentHash": "XZX2dwJaM0EmXUrbz1AET6GnXEMDorPlp9e7B3cQvyZG3eGs6Xmk//ZDybqbPuvPukbr9eOmFrsUxyrtiHm2LQ=="
+      },
+      "SonarAnalyzer.VisualBasic": {
+        "type": "Direct",
+        "requested": "[8.52.0.60960, )",
+        "resolved": "8.52.0.60960",
+        "contentHash": "BkdoTjU0UYl9OFQT2sLzoNFdSE1lmHFUjfO2If1gsEj4H90L1WeP6DAoP58NmwTAojRRuz10L56lP5CbCv6q3Q=="
+      },
```

Comment:
It seems weird these are being added to the package lock file now, when all we've done is change the version. Still, not sure there is much we can do about it.